src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1757,6 +1789,22 @@ export class SearchView extends ViewPane {
 		super.saveState();
 	}
 
+	private async retrieveFileStats(): Promise<void> {
+		const files = this.searchResult.matches().filter(f => !f.fileStat).map(f => f.resolveFileStat(this.fileService));
+		await Promise.all(files);
+	}
+
+	private async updateFileStats(elements: FileMatch[]): Promise<void> {
```

Comment:
Looks like it's updating all file stats, when just `elements` would be preferable